components/Banner/examples/supplemental/MDCBannerView.h

```diff
@@ -14,19 +14,33 @@
 
 #import <UIKit/UIKit.h>
 
-@class MDCButton;
+#import "MaterialButtons.h"
 
 __attribute__((objc_subclassing_restricted)) @interface MDCBannerView : UIView
 
-@property(nonatomic, copy, nonnull) NSString *text;
+/**
+ A view that displays the text on a @c MDCBannerView.
+ @discussion These properties of @c textLabel can be used to configure the text shown on @c MDCBannerView.
+ */
+@property(nonatomic, readonly, strong, nonnull) UILabel *textLabel;
 
-@property(nonatomic, strong, nullable) UIImage *image;  // default is nil
+/**
+ A view that displays the image on a @c MDCBannerView.
+ @discussion These properties of @c imageView can be used to configure the image shown on @c MDCBannerView.
+ If @c imageView 's own property @c image is @c nil, this view won't show on the @c MDCBannerView.
+ */
+@property(nonatomic, readonly, strong, nonnull) UIImageView *imageView;
 
-@property(nonatomic, assign) NSUInteger numberOfButtons;  // default is 1
+/**
+ A leading button that displays on a @c MDCBannerView.
+ @discussion This @c leadingButton displays on the leading edge of the view. If it can't occupy on the same line with @c trailingButton, it always shows above it.
```

Comment:
```suggestion
@c leadingButton is displayed on the leading edge of the view. If it does not fit on the same row as @c trailingButton, it will be placed above @c trailingButton.
```

Replaced some ambiguous `it`'s with direct names to clarify the intent.

Reply:
Thanks for the suggestion! It looks much more clear as a documentation:) Addressed in 5ed33d9c7280f7d2798dab0b2aad4651983c8238.